src/modules/Common/public/Start-SdnDataCollection.ps1

```diff
@@ -239,6 +243,13 @@ function Start-SdnDataCollection {
             Invoke-PSRemoteCommand -ComputerName $dataNodes -ScriptBlock {
                 [System.IO.FileInfo]$networkTraceDir = "$($using:workingDirectory)\NetworkTraces"
                 if (Test-Path -Path $networkTraceDir.FullName -PathType Container) {
+
+                    # convert the trace file into human readable format without requirement of additional parsing tools
+                    foreach ($file in (Get-ChildItem -Path "$($networkTraceDir.FullName)\*" -Include '*.etl')) {
```

Comment:
Should we limit to latest one as well here?

Reply:
Yes, there should only be one of these because the step after this performs a `Move-Item` operation, which moves this entire directory over and will not exist post data collection.

Reply:
Pushed changes as well to this area, to limit to single .etl in case there is multiple in this directory for any reason